app/javascript/packages/document-capture/components/selfie-capture.jsx

```diff
@@ -1,289 +0,0 @@
-import {
-  forwardRef,
-  useRef,
-  useState,
-  useEffect,
-  useCallback,
-  useContext,
-  useMemo,
-  useImperativeHandle,
-} from 'react';
-import { Icon } from '@18f/identity-components';
-import { useI18n } from '@18f/identity-react-i18n';
-import { useIfStillMounted, useInstanceId } from '@18f/identity-react-hooks';
-import FileImage from './file-image';
-import useFocusFallbackRef from '../hooks/use-focus-fallback-ref';
```

Comment:
This was the only usage of `useFocusFallbackRef`, so we can remove the hook too.